spec/index.md

```diff
@@ -29,4 +29,5 @@ This use case exports all resources needed to convey a patient's healththcare fi
 * [Overview Presentation](https://docs.google.com/presentation/d/14ZHmam9hwz6-SsCG1YqUIQnJ56bvSqEatebltgEVR6c/edit?usp=sharing)
 * [Client and Server Implementations](https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/implementations.md)
 * [Discussion Group (FHIR Zulip "Bulk Data" Track)](https://chat.fhir.org/#narrow/stream/bulk.20data)
-* [Argonaut Project: Bulk Data Export Security Risk Assessment Report](https://github.com/smart-on-fhir/fhir-bulk-data-docs/raw/9c5a9dcdfcd7b986f59a102bc2b4616552fed3e4/security-risk-assessment-report.pdf)
+* [Argonaut Project: Bulk Data Export Security Risk Assessment Report](
+  https://raw.githubusercontent.com/HL7/bulk-data/master/resources/documents/security-risk-assessment-report.pdf)
```

Comment:
This should not point to the master branch -- because that would cause published spec versions to point to incorpoarte mutable content. This should probably be a relative link, and we should search the entire spec for other instances of `bulk-data/master` to avoid them as well.

Reply:
@ricky1uphealth point to the relative url

Reply:
25fc279